bindings/matrix-sdk-ffi/src/api.udl

```diff
@@ -112,57 +106,179 @@ interface Room {
     [Throws=ClientError]
     string? member_display_name(string user_id);
 
-    BackwardsStream? start_live_event_listener();
-    void stop_live_event_listener();
+    void add_timeline_listener(TimelineListener listener);
+
+    // Loads older messages into the timeline.
+    //
+    // Raises an exception if there are no timeline listeners.
+    [Throws=ClientError]
+    PaginationOutcome paginate_backwards();
```

Comment:
The previous version of this method had a `u64 count` parameter. Would be be able to add that to this version too? Not necessarily now, just highlighting this as when the app is running on a Mac with a large display we would ideally want to paginate by a much larger number of events than when running on an old iPhone with a tiny screen.

Reply:
Sure.